Misc/NEWS.d/next/Library/2019-03-23-17-16-15.bpo-36407.LG3aC4.rst

```diff
@@ -0,0 +1 @@
+Fixed wrong indentation writing for CDATA section in xml.dom.minidom
```

Comment:
Add "Patch by *yourname*."

Reply:
Updated according to your recommendation